.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish PKI
+
+on:
+  workflow_run:
+    workflows: [ 'Build PKI' ]
+    branches:
+      - master
+    types:
+      - completed
```

Comment:
This workflow is triggered by any 'Build PKI' in the master. Is a good approach to publish the packages for all the pull requests even these could not work? Or we should limit to to the master push (after merge) so the package should be always valid?

Reply:
I'm not an expert in this area. Would this code run the publish job on pull request as well? I actually meant to run it on push only. How should we change that?

Reply:
As far as I know this run every time the trigger workflow ('Build PKI') is executed and in our case this is for both push and pull_request events. As an example, in the case of SonarCloud workflow, I have  added a condition to run only if the original event is a PR, it is in [this line](https://github.com/dogtagpki/pki/blob/ebf5e1100aab20d3f14f6fcc75031b01c00b8135/.github/workflows/sonarcloud-pull.yml#L10). I think you can modify the condition to work only for push.

Reply:
I tried creating a PR in my repo but it didn't seem to trigger the publish job (the publish job was already pushed to my master branch), maybe because of these lines:
```
on:
    ...
    branches:
      - master
```
Regardless, I added the line you suggested to ensure it only runs after a successful build job in the master branch. Thanks!

Reply:
From the documentation I was thinking that the branches specify the branch connected with the event but not the event so PR and push to master both should trigger this. Anyway, now it is limited to push in any case.